automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppLifecycleSpec.scala

```diff
@@ -65,12 +66,12 @@ class AppLifecycleSpec
     googleProject =>
       test(googleProject,
            createAppRequest(AppType.Allowed, "rstudio-test-workspace", None, Some(AllowedChartName.RStudio)),
-           true,
+           false,
            true
       )
   }
 
-  "create CUSTOM app, start/stop, delete it" taggedAs Retryable in { googleProject =>
+  "create CUSTOM app then delete it" taggedAs Retryable in { googleProject =>
```

Comment:
@LizBaldo do we need to test this anymore?

Reply:
Probably not, we are not supporting start/stop of GKE apps for now and so this would likely always fail?
That being said, the start/stop endpoints should maybe be commented out of the swagger page for the custom app users (they do not interact via terra ui, just the leo swagger page for now)